wagtail/contrib/simple_translation/forms.py

```diff
@@ -12,15 +12,33 @@ class CheckboxSelectMultipleWithDisabledOptions(forms.CheckboxSelectMultiple):
 
     def create_option(self, *args, **kwargs):
         option = super().create_option(*args, **kwargs)
+        # Disabling the option if it's in the disabled_values list
         if option["value"] in self.disabled_values:
             option["attrs"]["disabled"] = True
+        else:
+            # Note: only adding the target/action to the non-disabled ones so that
+            # disabled ones get ignored
+            option["attrs"]["data-action"] = "w-bulk#toggle"
+            option["attrs"]["data-w-bulk-target"] = "item"
         return option
 
 
 class SubmitTranslationForm(forms.Form):
     # Note: We don't actually use select_all in Python, it is just the
     # easiest way to add the widget to the form. It's controlled in JS.
-    select_all = forms.BooleanField(label=gettext_lazy("Select all"), required=False)
+
+    select_all = forms.BooleanField(
+        label=gettext_lazy("Select all"),
+        required=False,
+        widget=forms.CheckboxInput(
+            # note: we add the select all checkbox data attributes, we could do this in __init__ instead but seems to make sense here.
```

Comment:
```suggestion
```

This note was for the demo code only to explain this, not required for the actual real code.